README.md

```diff
@@ -1,91 +1,130 @@
+The main components of Determined are the Python library, the command line interface (CLI), and the Web UI.
 
- ```bash
+## Python Library
+Use the Python library to make your existing PyTorch or Tensorflow code compatible with Determined. 
 
-# Start a Determined cluster locally.
-python3 -m venv ~/.virtualenvs/test
-. ~/.virtualenvs/test/bin/activate
-pip install determined
-# To start a cluster with GPUs, remove `no-gpu` flag.
-det deploy local cluster-up --no-gpu
-# Access web UI at localhost:8080. By default, "determined" user accepts a blank password.
+You can do this by organizing your code into one of the class-based APIs:
+
+```python
+from determined.pytorch import PyTorchTrial
+
+class YourExperiment(PyTorchTrial):
+  def __init__(self, context):
+    ...
+```
+
+Or by using just the functions you want, via the Core API:
+
+```python
+import determined as det
+
+with det.core.init() as core_context:
+    ...
+```
+
+## Command Line Interface (CLI)
+
+Use the CLI to start the Determined cluster locally:
 
-# Navigate to a Determined example.
-git clone --recurse-submodules https://github.com/determined-ai/determined
-cd determined/examples/computer_vision/cifar10_pytorch
+```
+det deploy local cluster-up
+```
 
-# Submit job to train a single model on a single node.
-det experiment create const.yaml .
- ```
+Or on cloud services like AWS and GCP:
 
-## Detailed Installation Guide
+```
+det deploy aws up
+```
 
-See [our installation guide](https://docs.determined.ai/latest/how-to/install-main.html) for details on how to install Determined, including on AWS and GCP.
 
-### Try Now on AWS
+Then train your models:
+```bash
+det experiment create gpt.yaml .
+```
 
-[![Try Now](https://s3.amazonaws.com/cloudformation-examples/cloudformation-launch-stack.png)](https://console.aws.amazon.com/cloudformation/home?region=us-west-2#/stacks/create/review?templateURL=https://determined-ai-public.s3-us-west-2.amazonaws.com/simple.yaml)
+And use yaml files to configure everything from distributed training to hyperparameter tuning:
+
+```yaml
+resources:
+  slots_per_trial: 8
+  priority: 1
+hyperparameters:
+  learning_rate:
+    type: double
+    minval: .0001
+    maxval: 1.0
+searcher:
+  name: adaptive_asha
+  metric: validation_loss
+  smaller_is_better: true
+```
+
+
+## Web UI
+
+Use the Web UI to view loss curves, hyperparameter plots, code and configuration snapshots, model registries, cluster utilization, debugging logs, performance profiling reports, and more.
+
+![Web UI](docs/assets/readme_images/webui.png)
+
+
+# Installation
+
+Install the CLI:
+```bash
+pip install determined
+```
 
-## Next Steps
+Then use `det deploy` to start the Determined cluster locally, or on cloud services like AWS and GCP.
 
-For a brief introduction to using Determined, check out our
-[Quick Start Guide](https://docs.determined.ai/latest/getting-started.html).
+See the following guides for all installation details:
```

Comment:
For installation details, visit the the cluster deployment guide for your environment: